### PR TITLE
Answers -> Search 2: Rename imports

### PR DESCRIPTION
**Pull request by ElemelonWind**
J=SLAP-2230
TEST=auto

### CODE REVIEW

File: src/components/AppliedFiltersDisplay.tsx [hunk 1]
```diff
@@ -46,20 +46,20 @@ export function AppliedFiltersDisplay(props: AppliedFiltersDisplayProps): JSX.El
     hierarchicalFacetsDelimiter = DEFAULT_HIERARCHICAL_DELIMITER,
     cssClasses = {}
   } = props;
-  const answersActions = useAnswersActions();
-  const isVertical = useAnswersState(state => state.meta.searchType) === SearchTypeEnum.Vertical;
+  const SearchActions = useSearchActions();
```
Comment (nmanu1) [inline src/components/AppliedFiltersDisplay.tsx:49]: nit: can this be lowercase like it used to be (`searchActions`)? and similarly in other files

File: src/components/SearchBar.tsx [hunk 2]
```diff
@@ -180,23 +180,23 @@ export function SearchBar({
...
+    () => executeAutocompleteSearch(SearchActions)
   );
   const [
     executeQueryWithNearMeHandling,
     autocompletePromiseRef,
   ] = useSearchWithNearMeHandling(geolocationOptions, onSearch);
   const [recentSearches, setRecentSearch, clearRecentSearches] = useRecentSearches(recentSearchesLimit);
   const filteredRecentSearches = recentSearches?.filter(search =>
-    answersUtilities.isCloseMatch(search.query, query)
+    SearchUtilities.isCloseMatch(search.query, query)
```
Comment (alextaing) [inline src/components/SearchBar.tsx:199]: I think this `search` should also be lower case!

File: package.json [hunk 3]
```diff
@@ -133,6 +131,7 @@
     "@restart/ui": "^1.0.1",
     "@tailwindcss/forms": "^0.5.0",
     "@yext/analytics": "^0.2.0-beta.3",
+    "@yext/search-headless-react": "^1.3.0",
```
Comment (nmanu1) [inline package.json:134]: why was this changed to a regular dependency instead of dev and peer dependency?

File: src/components/AppliedFiltersDisplay.tsx (hunk 1 above)
Comment (ElemelonWind) [reply src/components/AppliedFiltersDisplay.tsx:49]: whoops, thanks! I'll change all of them back, I just used the replace function lol

File: package.json (hunk 3 above)
Comment (ElemelonWind) [reply package.json:134]: not sure how that happened, I'll take `package.json` from the feature branch and rename it again

**Comment by ElemelonWind** [comment]
Will fix the errors, forgot to change some of the variable names!

File: test-site/package.json [hunk 4]
```diff
@@ -43,4 +43,4 @@
       "last 1 safari version"
     ]
   }
-}
+}
```
Comment (nmanu1) [inline test-site/package.json:46]: nit: can you add this newline back. also can you revert the test-site `package-lock` changes

File: tests/__fixtures__/utils/location-operations.ts [hunk 5]
```diff
@@ -11,12 +11,12 @@ let userLocation: LatLong = {
 };
 
 export async function updateLocationIfNeeded(
-  answersActions: AnswersActions,
+  SearchActions: SearchActions,
```
Comment (nmanu1) [inline tests/__fixtures__/utils/location-operations.ts:14]: there are some more places in the tests where the variable name capitalization needs to be fixed

File: src/components/DirectAnswer.tsx [hunk 6]
```diff
@@ -59,8 +59,8 @@ const builtInCssClasses: Readonly<DirectAnswerCssClasses> = {
  * @returns A react element for DirectAnswer
```
Comment (nmanu1) [inline src/components/DirectAnswer.tsx:59]: nit: can you update the doc here to refer to "Search API"? and similarly for the `SpellCheck` component's docs

File: src/components/SearchBar.tsx [hunk 7]
```diff
@@ -110,7 +110,7 @@ export type RenderEntityPreviews = (
  */
 export interface VisualAutocompleteConfig {
   /** The Answers Headless instance used to perform visual autocomplete searches. */
```
Comment (nmanu1) [inline src/components/SearchBar.tsx:112]: nit: this doc should be updated to "Search Headless"

**Review by nmanu1 — COMMENTED** [review]
can you update the name of the repo in the LICENSE if it isn't already updated?

**Review by oshi97 — APPROVED** [review]

**Review by nmanu1 — COMMENTED** [review]
looks like there are still some changes in `package.json`, `package-lock.json`, and `test-site/package-lock.json` that need to be reverted

**Review by nmanu1 — APPROVED** [review]
🔥
